src/falkordb-sentinel/split-brain-monitor.sh

```
@@ -0,0 +1,292 @@
+        
+        # Check if all sentinels agree but are pointing to the wrong master
+        local all_sentinels_wrong=false
+        if [[ ${#unique_hosts[@]} -eq 1 && "${unique_hosts[0]}" != "$true_master" ]]; then
+            echo "$(date '+%Y-%m-%d %H:%M:%S') - All sentinels agree but pointing to wrong master!"
+            echo "Sentinels report: ${unique_hosts[0]}"
+            echo "True master: $true_master"
+            all_sentinels_wrong=true
+        fi
+        
+        # Check for split brain condition or all sentinels wrong
+        if [[ ${#unique_hosts[@]} -gt 1 || "$all_sentinels_wrong" == "true" ]]; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)